parse.go

```diff
@@ -228,6 +229,14 @@ func AllowMismatchPixelDataLength() ParseOption {
 	}
 }
 
+// AllowErrorMetaElementGroupLength allows parser to work around missing metaelement group length tag (0x0002,0x0000) by reading elements only
+// in group 2
```

Comment:
```suggestion
// in group 2.
```
nit: (should include punctuation, e.g. periods in go commentary: https://google.github.io/styleguide/go/decisions#comment-sentences)

Reply:
No problem glad to see great quality standards enforced.   Added period.